centreon/unattended.sh

```diff
@@ -281,29 +274,47 @@ function get_os_information() {
 # then concat the string for $CENTREON_REPO
 #
 function set_centreon_repos() {
+	if ! [ -z $1 ]; then
+		repo=$1
+	fi
 
 	IFS=', ' read -r -a array_repos <<<"$repo"
 
 	CENTREON_REPO=""
 	for _repo in "${array_repos[@]}"; do
 
 		[[ ! ${SUPPORTED_REPOSITORY[$_repo]} ]] &&
-			log "ERROR" "Unsupported repository : $_repo" &&
+			log "ERROR" "Unsupported repository: $_repo" &&
 			usage
 
-		CENTREON_REPO+="centreon-$_repo*"
+		CENTREON_REPO+="centreon-23.04-$_repo*"
 		if ! [ "$_repo" == "${array_repos[@]:(-1)}" ]; then
 			CENTREON_REPO+=","
 		fi
 	done
 
-	log "INFO" "Following Centreon repo will be used [$CENTREON_REPO]"
+	log "INFO" "Following Centreon repo will be used: [$CENTREON_REPO]"
 
 }
 #========= end of function set_centreon_repos()
 
+#========= begin of function set_mariadb_repos()
+#
+function set_mariadb_repos() {
+	log "INFO" "Install MariaDB repository"
+	curl -LsS https://r.mariadb.com/downloads/mariadb_repo_setup | sudo bash -s -- --mariadb-server-version="mariadb-10.5"
+	if [ $? -ne 0 ]; then
+		error_and_exit "Could not install the repository"
+	else
+		log "INFO" "Successfully installed MariaDB repository"
+	fi
+	rm -f -y /etc/yum.repos.d/mariadb.repo.* > /dev/null 2>&1
```

Comment:
why do you remove mariadb repo ?

Reply:
I remove copy of previous MariaDB repo if this one use to exist because this operation create a copy